tests/integration/relations/test_db.py

```diff
@@ -198,15 +197,3 @@ async def test_extensions_blocking(ops_test: OpsTest) -> None:
         raise_on_blocked=False,
         timeout=3000,
     )
-
-
-@pytest.mark.group(1)
-@pytest.mark.unstable
-async def test_relation_with_openldap(ops_test: OpsTest):
```

Comment:
The image for the charm cannot be downloaded.

Reply:
Which image? I deployed OpenLDAP through `juju deploy openldap-charmers-openldap --channel edge` and got it in the `Waiting for database relation` state.

Reply:
Looking deeper, it looks like this is the podspec charm and the pull failure seems to originate from [the image](https://git.launchpad.net/charm-k8s-openldap/tree/src/charm.py#n55) pulled by the podspec.